Content.Shared/Disposal/Components/DisposalUnitComponent.cs

```diff
@@ -7,8 +7,8 @@
 
 namespace Content.Shared.Disposal.Components;
 
-[NetworkedComponent]
-public abstract partial class SharedDisposalUnitComponent : Component
+[RegisterComponent, NetworkedComponent]
+public partial class DisposalUnitComponent : Component
```

Comment:
Same here.

Reply:
There is also no client / server stuff left to move here so I've just sealed it.